recipe/meta.yaml

```diff
@@ -5,35 +5,32 @@ package:
   version: {{ version }}
 
 source:
-  url: https://support.hdfgroup.org/ftp/HDF/releases/HDF{{ version }}/src/hdf-{{ version }}.tar.bz2
-  sha256: bde035ef5a1cd5fdbd0a7f1fa5c17e98bbd599300189ac4d234f16e9bb7bcb12
-  patches:
-    - patches/max_files.patch
-    - patches/0001-Don-t-build-ctest0-and-run-ncgen-when-not-testing.patch
-    - patches/debian/64bit.patch
-    - patches/debian/hdf-4.2.10-aarch64.patch
-    - patches/debian/hdfi.h.patch
-    - patches/debian/spelling-errors.patch
+  - git_url: https://github.com/HDFGroup/hdf4
```

Comment:
Out of curiosity, why are you using the git URL?

Reply:
1. The easiest way to test compiling several versions of the code without keeping track of lots of sha256 checksums.
2. Personal preference for dynamic over static code bases.